prisma/schema.prisma

```diff
@@ -11,11 +11,12 @@ datasource db {
 }
 
 model App {
-  id            Int       @id @default(autoincrement())
-  appId         String    @unique
-  appIcon       String
-  googlePlayUrl String
-  listings      Listing[]
+  id                    Int                     @id @default(autoincrement())
+  appId                 String                  @unique
+  appIcon               String
+  googlePlayUrl         String
+  listings              Listing[]
+  UserManagementRequest UserManagementRequest[]
```

Comment:
UserManagementRequest field should be:
* plural and begin with lowercase letter (like listings on App)
* it could be just "requests" or userManagementRequests"

Reply:
FYI, relationship names don't affect the migrations so you can change them without changing the database schema.